### PR TITLE
Simple carousel

### PR DESCRIPTION
**Pull request by MostafaRastegar**

### CODE REVIEW

File: packages/ui/packages/Carousel/src/core/utils.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable array-callback-return */
+/* eslint-disable radix */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-param-reassign */
+/* eslint-disable consistent-return */
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/src/core/utils.js]: combine this disables in one line, and do not disable every thing :|
please fix them, or at least do disable it for one line

File: packages/ui/packages/Carousel/test/utils.test.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/test/utils.test.js]: if you fixed the lint , why should disable it?

**Comment by sayjeyhi** [comment]
@all-contributors please add @MostafaRastegar  for ideas, tests and code

File: packages/ui/packages/Carousel/test/utils.test.js (hunk 2 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/test/utils.test.js]: I should install eslint-plugin-jest  for test lint, eslint not support without this plugin

Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/test/utils.test.js]: I add this plugin and fix

File: packages/ui/packages/Carousel/src/core/utils.js (hunk 1 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/src/core/utils.js]: done

**Pull request merged**